src/SetLanguageFromRoute.php

```diff
@@ -42,8 +42,10 @@ public function handle($request, Closure $next)
         if ($language === config('languages.default')) {
             $parts->shift(); // Chop off the language part
             $newPath = $parts->implode('/');
-            $queryString = http_build_query($request->query());
-            return redirect("{$newPath}?{$queryString}", 301);
+            if ($queryString = http_build_query($request->query())) {
```

Comment:
Is het volgende niet leesbaarder?

```php
$newPath = $parts->implode('/');
$queryString = http_build_query($request->query());
if ($queryString) {
    $newPath .= "?{$queryString}";
}
```

Reply:
Assignment in if-statements vind ik altijd een beetje tricky, je leest er snel overheen

Reply:
Ben er zelf eigenlijk wel fan van, gebruikt toch weer een regel minder. Ben het overigens wel met je eens dat het v.w.b. leesbaarheid niet heel handig is, zal het aanpassen.